packages/design/jobberStyle.js

```diff
@@ -2,6 +2,8 @@
 const fs = require("fs");
 // eslint-disable-next-line import/no-internal-modules
 const customPropertiesObject = require("./src/foundation.js");
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
It is a global atlantis rule and I don't think we want to disable it for most reasons because the bundle versions would resolve things incorrectly. However this file isn't actually released and is only used in the build process. The reason for not just disabling it on the file entirely is to make it noticable that the rule is being disabled but it may make sense for this file because it would only be used in the build set